security-header/Caddyfile

```diff
@@ -0,0 +1,10 @@
+header / {
+    Strict-Transport-Security "max-age=31536000; includeSubDomains; preload"
```

Comment:
Should probably mention on the readme a warning before site owners unknowingly shoot themselves in the foot (those who copy and paste without reading)...